hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -1522,6 +1522,19 @@
     </description>
 </property>
 
+<property>
+  <name>fs.s3a.metadatastore.fail.on.write.error</name>
+  <value>false</value>
```

Comment:
should change to `true` here

Reply:
oops, definitely, thanks

Reply:
fixed in https://github.com/apache/hadoop/pull/666/commits/aeccca13cbd5844dbfc486a899209cee048f8c52, will run tests again